src/language-js/print/type-annotation.js

```diff
@@ -287,15 +287,21 @@ function printFunctionType(path, options, print) {
 function printTupleType(path, options, print) {
   const node = path.getValue();
   const typesField = node.type === "TSTupleType" ? "elementTypes" : "types";
-  const hasRest =
-    node[typesField].length > 0 &&
-    getLast(node[typesField]).type === "TSRestType";
+  const types = node[typesField];
+  const isNotEmptyTuple = isNonEmptyArray(types);
```

Comment:
Let's use `isNonEmptyTuple` or just  `isNonEmpty` to keep  consistent with the function name?